assets/src/components/detours/diversionPanel.tsx

```diff
@@ -34,9 +36,19 @@ export const DiversionPanel = ({
 
     <Panel.Body className="c-diversion-panel__body d-flex flex-column">
       <Panel.Body.ScrollArea>
+        {inTestGroup(TestGroups.RouteLadderHeaderUpdate) && (
+          <Button
+            className="align-self-start icon-link my-3"
```

Comment:
suggestion: this doesn't seem to be doing anything?
```suggestion
            className="icon-link my-3"
```

Reply:
I think the difference is that `DetourFinishedPanel` uses `d-flex`